test_resources/kinbody/objects/smallsphere.kinbody.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<KinBody>
+	<Body type="dynamic">
+		<Geom type="sphere">
+			<Translation>0  0  0</Translation>
+			<diffuseColor>0 0.5 0</diffuseColor>
+			<radius>0.013</radius>
+		</Geom>
+	</Body>
+</KinBody>
```

Comment:
Nit: Change this file to use the same spacing as the other `.kinbody.xml` files.